pallets/x-transfer/src/lib.rs

```diff
@@ -0,0 +1,58 @@
+// Copyright 2019-2020 PureStake Inc.
+// This file is part of Moonbeam.
+
+// Moonbeam is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Moonbeam is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Moonbeam.  If not, see <http://www.gnu.org/licenses/>.
+
+//! Minimal Pallet that stores the numeric Ethereum-style chain id in the runtime.
```

Comment:
:spaghetti:

Reply:
This is the one that will impl DownwardMessageHandler, right?

Reply:
Ye, I'll probably rename it to `token-dealer` so it just replaces that existing pallet.